pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -443,11 +464,13 @@ func launchScanForSuite(r *ReconcileComplianceSuite, suite *compv1alpha1.Complia
 	return nil
 }
 
-func newScanForSuite(suite *compv1alpha1.ComplianceSuite, scanWrap *compv1alpha1.ComplianceScanSpecWrapper) *compv1alpha1.ComplianceScan {
+func newScanForSuite(suite *compv1alpha1.ComplianceSuite, scanWrap *compv1alpha1.ComplianceScanSpecWrapper, profileUniqueID string) *compv1alpha1.ComplianceScan {
 	scan := compv1alpha1.ComplianceScanFromWrapper(scanWrap)
 	scan.SetLabels(map[string]string{
-		compv1alpha1.SuiteLabel: suite.Name,
+		compv1alpha1.SuiteLabel:       suite.Name,
+		compv1alpha1.ProfileGuidLabel: profileUniqueID,
```

Comment:
Same comment here as above. I wonder if we should be setting this to `"compliance.openshift.io/profile-guid"` so that it's more clear this ID isn't the ComplianceScan GUID.